open-widget-framework/src/widget-list.js

```diff
@@ -11,6 +11,9 @@ class WidgetList extends Component {
   * Props:
   *    fetchRoute: where to fetch widgets in list from
   */
+
+  static defaultProps = {...defaultSettings}
```

Comment:
This is just so that the defaults are accessible? They're not actually being called anywhere in your rework correct?